### PR TITLE
Handle new columns.

### PR DESCRIPTION
**Pull request by geekbeast**
This adds new columns and updates download processing for time use diary data.

### CODE REVIEW

File: src/main/kotlin/com/openlattice/chronicle/services/timeusediary/TimeUseDiaryService.kt [hunk 1]
```diff
@@ -302,34 +310,110 @@ class TimeUseDiaryService(
...
+        val activityDayStartDateTime = LocalTime.parse(activityDayStartTime).atDate(activityDate))
+        val activityDayEndDateTime = LocalTime.parse(activityDayEndTime).atDate(activityDate)
+        val bedTimeBeforeActivityDayDateTime = if (bedTimeBeforeActivityDay != null) {
+            LocalTime.parse(bedTimeBeforeActivityDay)
+                .atDate(activityDate.minusDays(1))
+                .atZone(zoneIdOfPrimaryActivity)
+        } else null
+        val wakeUpTimeAfterActivityDayDateTime = if (wakeUpTimeAfterActivityDay != null) {
+            LocalTime.parse(wakeUpTimeAfterActivityDay)
+                .atDate(activityDate.minusDays(1))
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/chronicle/services/timeusediary/TimeUseDiaryService.kt]: This should be plusDays

File: src/main/kotlin/com/openlattice/chronicle/services/timeusediary/TimeUseDiaryService.kt [hunk 2]
```diff
@@ -302,34 +310,110 @@ class TimeUseDiaryService(
...
+        } else null
+
+        //This bug remains for compatibility with legacy downloads.
+        val todayWakeUpDateTime = if (todayWakeUpTime != null) {
+            LocalTime
+                .parse(todayWakeUpTime).atDate(LocalDate.now())
+                .atZone(zoneIdOfPrimaryActivity)
+        } else null
+
+        val confusing = if (todayWakeUpDateTime != null) {
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/chronicle/services/timeusediary/TimeUseDiaryService.kt]: Rename confusing variable

File: src/main/kotlin/com/openlattice/chronicle/services/timeusediary/TimeUseDiaryService.kt [hunk 3]
```diff
@@ -302,34 +310,110 @@ class TimeUseDiaryService(
...
+            mapOf(
+                TimeUseDiaryColumTitles.WAKE_UP_TODAY to (todayWakeUpDateTime?.toLocalTime()?.format(formatter) ?: ""),
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/chronicle/services/timeusediary/TimeUseDiaryService.kt]: Null checks aren't necessary hear due to if statement.

**Review by geekbeast — COMMENTED** [review]
Apparently, I can't request changes on my own PR.

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**